contracts/truefi2/LoanFactory2.sol

```diff
@@ -89,8 +89,8 @@ contract LoanFactory2 is ILoanFactory2, Initializable {
     ) internal view returns (uint256) {
         uint8 borrowerScore = creditOracle.score(borrower);
         uint256 fixedTermLoanAdjustment = rateAdjuster.fixedTermLoanAdjustment(_term);
-        uint256 proFormaRate = rateAdjuster.proFormaRate(pool, borrowerScore, amount);
-        return proFormaRate.add(fixedTermLoanAdjustment);
+        uint256 Rate = rateAdjuster.Rate(pool, borrowerScore, amount);
```

Comment:
Why a capital letter?

Reply:
grep's too easy :)

Don't forget to uncapitalize all other function names too.